### PR TITLE
ref(ingest): Emit outcomes for standalone attachments

### PR DESCRIPTION
**Pull request by jan-auer**
Also emits outcomes for individual attachments sent to the attachments endpoint in Relay. This allows to track and bill for individual attachment uploads that happen in separate requests.

None of our SDKs do this, but it is theoretically possible with Envelopes.

### CODE REVIEW

**Review by untitaker — APPROVED** [review]
rubberstamping for now, looks fine on first glance. doing closer review later

File: src/sentry/event_manager.py [hunk 1]
```diff
@@ -1271,48 +1346,14 @@ def save_attachments(cache_key, attachments, job):
...
-            file=file,
-        )
-
-        track_outcome(
-            org_id=event.project.organization_id,
-            project_id=job["project_id"],
+        save_attachment(
+            cache_key,
+            attachment,
+            event.project,
```
Comment (untitaker) [inline src/sentry/event_manager.py:1352]: This attribute access may hit the database. Please verify `event.project` is somehow fetched from cache first or pass `event.project_id` like the old code.

Comment (untitaker) [reply src/sentry/event_manager.py:1352]: Alternatively edit the `Event` class to fetch from cache for this property. That's probably a good idea in all cases.

**Review by untitaker — APPROVED** [review]
ok this is pretty straightforward. found one performance concern

File: src/sentry/event_manager.py (hunk 1 above)
Comment (jan-auer) [reply src/sentry/event_manager.py:1352]: We did `event.project.organization_id` before (see [here](https://github.com/getsentry/sentry/pull/19787/files#diff-4bf345e0795c0e86cfdb6a13f60bcb64L1278)), and there are multiple code paths that already access project. In fact, the project is guaranteed to be bound at this time. 

Good catch though!

**Pull request merged**